src/wmtk/Mesh.cpp

```diff
@@ -119,6 +119,20 @@ long Mesh::capacity(PrimitiveType type) const
     return m_attribute_manager.m_capacities.at(get_primitive_type_id(type));
 }
 
+
+bool Mesh::is_boundary(const Tuple& tuple) const
+{
+    long my_dim = top_cell_dimension() - 1;
```

Comment:
`const` (I know it's 3 lines of code and it doesn't really matter here but I still think it's a good habit)